openedx/stanford/cms/envs/test.py

```diff
@@ -5,8 +5,8 @@
 INSTALLED_APPS += (
     'openedx.stanford.djangoapps.register_cme',
 )
-# Remove sneakpeek during tests to prevent unwanted redirect
+
 MIDDLEWARE_CLASSES = tuple([
-    mwc for mwc in MIDDLEWARE_CLASSES
-    if mwc != 'sneakpeek.middleware.SneakPeekLogoutMiddleware'
+    c for c in MIDDLEWARE_CLASSES
```

Comment:
Pylint usually expects variables to be at least 3 chars; let's revert the variable rename.